src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,225 @@
+import axios from 'axios'
+import shared from 'botpress/shared'
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+export interface StoreInfoBody {
+  storedInfo: string
+  loadInfo: Function
+}
+
+export interface StoreInfoType {
+  [key: string]: StoreInfoBody
+}
+
+export const storeInfos: StoreInfoType = {}
+
+export interface EventPackageInfoType {
+  [key: string]: EventPackageInfoBody
+}
+
+export interface EventPackageInfoBody {
+  locked: boolean
+  timeout: string
+  getPackage: Function
+}
+
+export const eventPackageInfo: EventPackageInfoType = {}
```

Comment:
Should I use instead something like a class? If not would there be anything better because with how the function works I don't think that anything else would work

Reply:
The way you coded it yes, I feel like your code is made to used objects but you didn't use any